src/components/GroupLinks.js

```diff
@@ -4,7 +4,11 @@ import PropTypes from "prop-types";
 const GroupLinks = ({ Website, Facebook, Twitter, Meetup }) => (
   <div className="links">
     {Website && (
-      <a href={Website} target="group" rel="nofollow" className="fas fa-link" />
+      <a href={Website}
+        target="group"
+        rel="nofollow"
+        className="fas fa-link"
+      />
```

Comment:
Ha, no worries! It can be a little odd sometimes, because sometimes bits of code can seem inconsistent with similar bits, but it's helpful to have the code consistent across all the changes we make.